File: MORM.Java/src/MORM/Java/Models/Venda/Transitem.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "TRANSITEM")
public class Transitem extends CollectionItem
{
    @CampoAttribute(Campo = "ID_TRANSACAO", Tipo = CampoTipo.Key)
    public String Id_Transacao;
    @CampoAttribute(Campo = "NR_ITEM", Tipo = CampoTipo.Key)
    public Integer Nr_Item;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "ID_PRODUTO", Tipo = CampoTipo.Req)
    public String Id_Produto;
    @CampoAttribute(Campo = "CD_PRODUTO", Tipo = CampoTipo.Req)
    public Integer Cd_Produto;
    @CampoAttribute(Campo = "DS_PRODUTO", Tipo = CampoTipo.Req)
    public String Ds_Produto;
    @CampoAttribute(Campo = "CD_CFOP", Tipo = CampoTipo.Req)
    public Integer Cd_Cfop;
    @CampoAttribute(Campo = "CD_ESPECIE", Tipo = CampoTipo.Req)
    public String Cd_Especie;
    @CampoAttribute(Campo = "CD_NCM", Tipo = CampoTipo.Req)
    public String Cd_Ncm;
    @CampoAttribute(Campo = "QT_ITEM", Tipo = CampoTipo.Req)
    public Float Qt_Item;
    @CampoAttribute(Campo = "VL_CUSTO", Tipo = CampoTipo.Req)
    public Float Vl_Custo;
    @CampoAttribute(Campo = "VL_UNITARIO", Tipo = CampoTipo.Req)
    public Float Vl_Unitario;
    @CampoAttribute(Campo = "VL_ITEM", Tipo = CampoTipo.Req)
    public Float Vl_Item;
    @CampoAttribute(Campo = "VL_VARIACAO", Tipo = CampoTipo.Req)
    public Float Vl_Variacao;
    @CampoAttribute(Campo = "VL_VARIACAOCAPA", Tipo = CampoTipo.Req)
    public Float Vl_Variacaocapa;
    @CampoAttribute(Campo = "VL_FRETE", Tipo = CampoTipo.Req)
    public Float Vl_Frete;
    @CampoAttribute(Campo = "VL_SEGURO", Tipo = CampoTipo.Req)
    public Float Vl_Seguro;
    @CampoAttribute(Campo = "VL_OUTRO", Tipo = CampoTipo.Req)
    public Float Vl_Outro;
    @CampoAttribute(Campo = "VL_DESPESA", Tipo = CampoTipo.Req)
    public Float Vl_Despesa;
}

File: MORM.CSharp/MORM.Dominio/Models/Venda/Transitem.cs
using MORM.Util.Atributos;
using MORM.Util.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("TRANSITEM")]
    public class Transitem : CollectionItem
    {
        [Campo("ID_TRANSACAO", CampoTipo.Key)]
        public string Id_Transacao { get; set; }
        [Campo("NR_ITEM", CampoTipo.Key)]
        public int Nr_Item { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("ID_PRODUTO", CampoTipo.Req)]
        public string Id_Produto { get; set; }
        [Campo("CD_PRODUTO", CampoTipo.Req)]
        public int Cd_Produto { get; set; }
        [Campo("DS_PRODUTO", CampoTipo.Req)]
        public string Ds_Produto { get; set; }
        [Campo("CD_CFOP", CampoTipo.Req)]
        public int Cd_Cfop { get; set; }
        [Campo("CD_ESPECIE", CampoTipo.Req)]
        public string Cd_Especie { get; set; }
        [Campo("CD_NCM", CampoTipo.Req)]
        public string Cd_Ncm { get; set; }
        [Campo("QT_ITEM", CampoTipo.Req)]
        public double Qt_Item { get; set; }
        [Campo("VL_CUSTO", CampoTipo.Req)]
        public double Vl_Custo { get; set; }
        [Campo("VL_UNITARIO", CampoTipo.Req)]
        public double Vl_Unitario { get; set; }
        [Campo("VL_ITEM", CampoTipo.Req)]
        public double Vl_Item { get; set; }
        [Campo("VL_VARIACAO", CampoTipo.Req)]
        public double Vl_Variacao { get; set; }
        [Campo("VL_VARIACAOCAPA", CampoTipo.Req)]
        public double Vl_Variacaocapa { get; set; }
        [Campo("VL_FRETE", CampoTipo.Req)]
        public double Vl_Frete { get; set; }
        [Campo("VL_SEGURO", CampoTipo.Req)]
        public double Vl_Seguro { get; set; }
        [Campo("VL_OUTRO", CampoTipo.Req)]
        public double Vl_Outro { get; set; }
        [Campo("VL_DESPESA", CampoTipo.Req)]
        public double Vl_Despesa { get; set; }
    }
}